package.json

```diff
@@ -41,6 +42,8 @@
     "bs-platform": "^5.0.4",
     "file-loader": "^3.0.1",
     "html-webpack-plugin": "^3.2.0",
+    "patch-package": "^6.1.2",
+    "postinstall-postinstall": "^2.0.0",
```

Comment:
is this needed?

Reply:
@eliabejr 	is patching a package. What other post install method you would recommend @fakenickels 	?

Reply:
ah no I just mean why to have this package that does postinstall

Reply:
yeah so what do you recommend to do postinstall? haha

Reply:
ahh, just followed the steps to install, can I remove it?

Reply:
>yeah so what do you recommend to do postinstall? haha


ue because npm already has a postinstall!

Reply:
>ahh, just followed the steps to install, can I remove it?

aah got it! steps from patch-package?

Reply:
yes!